openapi.yaml

```diff
@@ -0,0 +1,268 @@
+                $ref: "#/components/schemas/EmptyObject"
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        '403':
+          description: Forbidden.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /signals:
+    get:
+      summary: Get signals for a fic.
+      operationId: get_signals
+      tags:
+        - signals
+      security:
+        - cookieAuth: []
+      parameters:
+        - name: url
+          in: query
+          required: true
+          description: The URL of the fic to retrieve signals for.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Expected response to a valid request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Signals"
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        '403':
+          description: Forbidden.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    patch:
+      summary: Update signals for a fic.
+      operationId: patch_signals
+      tags:
+        - signals
+      security:
+        - cookieAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PatchSignalsRequest'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EmptyObject"
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        '403':
+          description: Forbidden.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+components:
+  securitySchemes:
+    cookieAuth:
+      type: apiKey
+      in: cookie
+      name: FicAiSession
+  schemas:
+    EmptyObject:
+      description: Empty object response.
+      type: object
+    Error:
+      description: |
+        Details about the unsuccessful fulfillment of a request.
+
+        Refer to the http status for a machine readable error code.
+      type: object
+      required:
+        - error
+      properties:
+        error:
+          type: object
+          required:
+            - message
+          properties:
+            message:
+              description: Human readable error message.
+              type: string
+              example: 'bad request'
+    CreateAccountRequest:
```

Comment:
It's called `CreateAccountQ` in server code, maybe use the same name here?

Reply:
Changed.